PXB-2.4-2666 Azure changes

rahulmalik87 opened the pull request:
https://jira.percona.com/browse/PXB-2666
Follow up fix PXB-1883 support for azure storage in xbcloud
https://jira.percona.com/browse/PXB-1883

Addded environment variable for storage class.
Rename the ENDPOINT environment variable to AZURE_ENDPOINT

altmannmarcelo reviewed (approved):
LGTM, please fix 8.0 PR .
1) Make sure you are merging this single commit into the 8.0 PR
2) Currently 8.0 has 3 commits, please make sure it has only 2

Commit:
PXB-2666 Azure changes
https://jira.percona.com/browse/PXB-2666

Follow up fix PXB-1883 support for azure storage in xbcloud
https://jira.percona.com/browse/PXB-1883
Addded environment variable for storage class.
Rename the ENDPOINT environment variable to AZURE_ENDPOINT
